.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+  - repo: https://github.com/renovatebot/pre-commit-hooks
+    rev: 36.40.3
+    hooks:
+      - id: renovate-config-validator
```

Comment:
Does this automatically discover all renovate configs?

Reply:
Yup `'(^|/).?renovate(?:rc)?(?:\.json5?)?$'`, I also manually tested it found both `renovate.json` and `infra/terraform/test-org/github/resources/renovate.json`